docs/feature-store/sources-targets.md

```diff
@@ -27,6 +27,28 @@ You can also create a custom `source` to access various databases or data source
 | [Kafka source](#kafka-source)                                                  |Event-based. Sets a Kafka source for the flow (supports both Apache and Confluence Kafka).| Y      | N     | N      |
 | {py:meth}`~mlrun.datastore.StreamSource`                                                       |Event-based. Sets the stream source for the flow. If the stream doesn’t exist it creates it. | Y      | N     | N      |
 
+## Snowflake source
+```{admonition} Note
+# Snowflake SQL columns are always set to uppercase letters.
+# This code will fail because timestamp_key is in lowercase:
+
+os.environ["SNOWFLAKE_PASSWORD"] = "*****"
```

Comment:
Better give a description and  working example and then add a note about uppercase.